src/payform.coffee

```diff
@@ -54,6 +54,16 @@
 
   payform = {}
 
+  # Key Codes
+  
+  keyCodes = {
+    UNKNOWN : 0,
+    BACKSPACE : 8,
+    PAGE_UP : 33,
```

Comment:
Can we add HOME and END? I feel that those are also important keys to make sure still work. Really liked the mapping - much more legible!